Fix-navigation-logo-when-switching-to-mobile

Cimlah opened the pull request:
On mobile the header always show the large logo.
Fixed this issue based on `isSidebarCollapsed` prop and `viewportWidth`

Cimlah committed:
Removed commented component, which I forgot to remove previously

Cimlah committed:
On mobile the header always show the large logo.
Fixed this issue based on `isSidebarCollapsed` prop and `viewportWidth`

mentor-tara reviewed (approved):
Go go go :smile:[](prolog-5)

Cimlah committed:
Added setting `viewportWidth` at the beggining.
Without it, the `viewportWidth` wasn't initially set and therefore
logo and sidebar texts weren't rendered properly on desktop

mentor-tara reviewed (approved):
Go go go :smile:[](prolog-5)

Cimlah commented:
Previous behaviour: When a user switches device orientation in a way that results in a change from desktop to mobile mode the small logo is shown instead of the large one.
Expected bahaviour: On mobile devices, the header should always show the large logo.

Fixed 👍
